src/pd/retry.rs

```diff
@@ -59,6 +59,8 @@ macro_rules! retry {
                 Ok(r) => return Ok(r),
                 Err(e) => last_err = Err(e),
             }
+            
+            drop($cluster);
```

Comment:
I think this needs a comment because changing it will only cause an error at runtime, so it's not obvious why it is here. I think it might be cleared to use a block rather than an explicit `drop`, but there should be a comment either way

Reply:
It was intended to resolve a lifetime problem on block style. And I've figured out a solution, so no drop will be needed.